script/prerequisitesForGetEdgeTPUToolchain.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# This shell script add PPA to get edgetpu-compiler package for EdgeTPUToolchain.
+
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+echo "deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | sudo tee /etc/apt/sources.list.d/coral-edgetpu.list
```

Comment:
```suggestion
echo "deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | tee /etc/apt/sources.list.d/coral-edgetpu.list
```

Reply:
This script is executed with root permission from ONE-vscode. Therefore, there is no need to add `sudo` command in this script.